src/libs/modulegroups.c

```diff
@@ -1773,6 +1773,9 @@ void init_presets(dt_lib_module_t *self)
 
   dt_lib_presets_add(_("workflow: scene-referred"), self->plugin_name, self->version(), tx, strlen(tx), TRUE);
 
+#if 0
```

Comment:
I have not tested, but reading the code, I fear that you remove the "search only" preset along with the "deprecated" one here... is that on purpose ? (or maybe I'm completely wrong  ;))

Reply:
No, not on purpose... Let me check!

Reply:
You were right, now fixed.